.github/workflows/goLint.yml

```
@@ -66,7 +70,7 @@ jobs:
           args: '${{ inputs.golangci-lint-args }}'
           skip-cache: true
       - name: Run go generate
-        if: success() || failure() # run this step even if the previous one failed
+        if: ( success() || failure() ) && !inputs.skip-go-generate
```

Does this work or do we need to use `${{ ... }}`?
```suggestion
        if: ${{ (success() || failure()) && !inputs.skip-go-generate }}
```

---

`if` doesn't need the `${{ ... }}`.